collectors/apps.plugin/apps_groups.conf

```diff
@@ -182,7 +183,7 @@ heapster: heapster
 # -----------------------------------------------------------------------------
 # AWS
 
-aws-s3: '*aws s3*'
+aws-s3: '*aws s3*' s3cmd s5cmd
```

Comment:
should we change the group name? It is `aws-s3`, but we have s**5**cmd?

Reply:
Nope, Amazon S3 is API, s**3**cmd is a old_slow_py client, s**5**cmd is a modern_fast_golang client (see [benchmarks](https://github.com/peak/s5cmd#benchmarks)). Why **5**? Becasue also we have a [s**4**cmd](https://github.com/bloomreach/s4cmd)!

Reply:
Thanks! My experience with AWS is only buying items on Amazon UK 😄